app/[locale]/header.tsx

```diff
@@ -126,11 +127,41 @@ export default async function Header({ locale }: { locale: string }) {
               <Link href={`/${locale}/login`}>{text.login}</Link>
             </Button>
           </li>
-          <li className="lg:hidden">
-            <IoMenu
-              className="rounded-md bg-primary-700 p-1 text-shade-light hover:cursor-pointer"
-              size={40}
-            />
+          <li className="z-30  font-semibold lg:hidden">
+            <nav className="relative flex h-0">
+              <MobNavButton
+                className="peer sticky z-40 h-10 w-10 rounded bg-primary-900 transition-colors aria-expanded:bg-transparent"
+                data-dropdownignore={true}
+              />
+              <aside
+                className="absolute -right-2 -top-2 flex w-80 max-w-[calc(100vw-1rem)] transform cursor-default flex-col gap-y-4 rounded-md border border-primary-500 bg-background p-6 opacity-0 transition-opacity peer-aria-expanded:opacity-100"
+                data-dropdownignore={true}
+              >
+                <ul
+                  className="space-y-4 text-base font-semibold"
+                  data-dropdownignore={true}
+                >
+                  {items.map(({ label, href }, index) => (
+                    <li key={index} className="w-fit">
+                      <Button
+                        asChild
+                        className="text-left text-shade-dark"
+                        variant="link"
+                      >
+                        <Link href={`/${locale}/${href}`}>{label}</Link>
+                      </Button>
+                    </li>
+                  ))}
+                </ul>
+                <hr className="opacity-50" data-dropdownignore={true} />
```

Comment:
This should also work
```suggestion
                <hr className="opacity-50" data-dropdownignore />
```